### PR TITLE
Expose disk IO read/write to Stackdriver sink

### PR DESCRIPTION
**Pull request by wangzhen127**
This PR expose disk IO read/write to Stackdriver sink.

### CODE REVIEW

File: metrics/core/labels.go [hunk 1]
```diff
@@ -105,6 +105,10 @@ var (
 		Key:         "accelerator_id",
 		Description: "ID of the accelerator",
 	}
+	LabelDiskName = LabelDescriptor{
+		Key:         "disk_name",
```
Comment (dashpole) [inline metrics/core/labels.go]: we probably want this to be "device"?

File: metrics/core/metrics.go [hunk 2]
```diff
@@ -1008,17 +1008,17 @@ var MetricDiskIORead = Metric{
...
 		return spec.HasDiskIo
 	},
 	GetLabeledMetric: func(spec *cadvisor.ContainerSpec, stat *cadvisor.ContainerStats) []LabeledMetric {
 		result := make([]LabeledMetric, 0, len(stat.DiskIo.IoServiceBytes))
 		for _, ioServiceBytesPerPartition := range stat.DiskIo.IoServiceBytes {
-			resourceIDKey := ioServiceBytesPerPartition.Device
-			if resourceIDKey == "" {
-				resourceIDKey = fmt.Sprintf("%d:%d", ioServiceBytesPerPartition.Major, ioServiceBytesPerPartition.Minor)
+			diskName := ioServiceBytesPerPartition.Device
+			if diskName != DevSda {
```
Comment (dashpole) [inline metrics/core/metrics.go]: can we confirm that this is the same on ubuntu?

**Review by dashpole — COMMENTED** [review]
looks good overall.  A couple nits

File: metrics/core/labels.go (hunk 1 above)
Comment (wangzhen127) [reply metrics/core/labels.go]: Done

File: metrics/core/metrics.go (hunk 2 above)
Comment (wangzhen127) [reply metrics/core/metrics.go]: Verified.

**Comment by dashpole** [comment]
This /lgtm

**Comment by wangzhen127** [comment]
Fixed a few nits from `make test-unit`.

**Review by bmoyles0117 — APPROVED** [review]

**Comment by bmoyles0117** [comment]
Can you please rebase off of master and/or resolve conflicts?

**Comment by wangzhen127** [comment]
I just rebased it. PTAL.

**Comment by wangzhen127** [comment]
I just updated the PR according to the new Stackdriver review. PTAL.

**Comment by wangzhen127** [comment]
Ping :)

**Comment by wangzhen127** [comment]
Dropping this now.

**Pull request closed without merging**